### PR TITLE
Apply new style in login page.

### PR DESCRIPTION
**Pull request by ace0625**

### CODE REVIEW

File: app/components/signIn/styles.js [hunk 1]
```diff
@@ -4,24 +4,119 @@ import {
 } from 'react-native';
 
 const WIDTH = Dimensions.get('window').width;
-const HEIGHT = Dimensions.get('window').height;
+export const HEIGHT = Dimensions.get('window').height;
```
Comment (SarahhhC) [inline app/components/signIn/styles.js]: Don't need to export.

File: app/app.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ import Crop from './components/camera/crop';
 import FlipCardDetailView from './components/detailView/flipCard';
 import Expanded from './components/expanded/expanded';
 import SearchBar from './components/search/search';
+import FirstPage from './components/signIn/first';
```
Comment (SarahhhC) [inline app/app.js:22]: Why we need first page?

Comment (SarahhhC) [reply app/app.js:22]: Start with Login page is more simple I think...

**Review by SarahhhC — COMMENTED** [review]
LGTM, Check some comments!

**Comment by SarahhhC** [comment]
Test webhook

File: app/app.js (hunk 2 above)
Comment (ace0625) [reply app/app.js:22]: That's for the router to register.

Comment (SarahhhC) [reply app/app.js:22]: Ok understood!

**Comment by SarahhhC** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/recbook/recbook-client/pull-request/41/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=recbook/recbook-client)

**Pull request merged**